classes/publication/PKPPublication.inc.php

```diff
@@ -145,17 +145,28 @@ public function getTitles()
      * Eg - Daniel Barnes, Carlo Corino (Author); Alan Mwandenga (Translator)
      *
      * @param array $userGroups List of UserGroup objects
+     * @param bool $includeInBrowseOnly true if only the includeInBrowse Authors will be contained
      *
      * @return string
      */
-    public function getAuthorString($userGroups)
+    public function getAuthorString($userGroups, $includeInBrowseOnly = false)
     {
         $authors = $this->getData('authors');
 
         if (empty($authors)) {
             return '';
         }
 
+        if ($includeInBrowseOnly) {
+            $authors = $authors->filter(function ($author, $key) {
+                return $author->getData('includeInBrowse');
+            });
+
+            $authors = $authors->sortBy(function ($author, $key) {
+                return $author->getData('seq');
+            });
```

Comment:
I think sorting needs to occur whether or not `$includeInBrowseOnly` is passed.

Reply:
This could be deprecated due to code changes.